deploy/helm/magda/charts/registry-api/templates/_helpers.tpl

```diff
@@ -51,14 +51,21 @@ spec:
 {{- if .root.Values.global.enableLivenessProbes }}
         livenessProbe:
           httpGet:
-            path: "/v0/ping"
+            path: /v0/status/live
             port: 80
-          initialDelaySeconds: 60
+          initialDelaySeconds: 10
           periodSeconds: 10
-          timeoutSeconds: {{ .root.Values.livenessProbe.timeoutSeconds | default 10 }}
```

Comment:
Is there a reason why we don't need this anymore? The reason this got put in is because the registry tends to slow down over time, so to mitigate it we made it restart when the liveness probe went over 5 seconds in prod, until https://github.com/magda-io/magda/issues/749 is done.